src/hisp/festim_models/mb_model.py

```diff
@@ -538,8 +540,8 @@ def make_B_mb_model(
 
     ############# Settings #############
     my_model.settings = F.Settings(
-        atol=1e-15,
-        rtol=1e-15,
+        atol=1e10,
+        rtol=1e-5,
```

Comment:
can we try 1e-10 for `rtol`?

Reply:
was getting really bad results with 1e-10 -- i can try something between?

Reply:
I don't see why this would give bad results ... let's keep it like that and make a note of it

Reply:
lowest i can get it with good results is 1e-8